src/mcts/search.cc

```diff
@@ -86,6 +87,7 @@ void Search::PopulateUciParams(OptionsParser* options) {
                             "policy-softmax-temp") = 1.0f;
   options->Add<IntOption>(kAllowedNodeCollisionsStr, 0, 1024,
                           "allowed-node-collisions") = 0;
+  options->Add<BoolOption>(kIgnoreNonmateStr, "ignore-nonmate") = false;
```

Comment:
"Ignore nonmate = false" is a triple negative!
Maybe come up with some "shiny" parameter name?
Like.. "sticky-checkmate" or at least maybe "visit-mate-alternatives" (=true by default)

Reply:
I was definitely not so sure about the name :) Either of those are fine by me, I guess I lean towards the shorter one by default